sdk/pineconevectordb/Azure.ResourceManager.PineconeVectorDB/src/Generated/ArmPineconeVectorDBModelFactory.cs

```diff
@@ -49,6 +49,7 @@ public static PineconeVectorDBOrganizationData PineconeVectorDBOrganizationData(
         /// <param name="provisioningState"> Provisioning state of the resource. </param>
         /// <param name="partnerDisplayName"> Pinecone Organization Name. </param>
         /// <param name="singleSignOnProperties"> Single sign-on properties. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="partnerDisplayName"/> is null. </exception>
```

Comment:
this looks incorrect, because our model factory would not throw this exception

Reply:
This should not be a blocker for now, filed https://github.com/Azure/azure-sdk-for-net/issues/53445 for tracking